R/CiteSeq.R

```diff
@@ -459,11 +459,14 @@ ProcessCiteSeqData <- function(seuratObj, assayName = 'ADT'){
 
 	#Compare new/old:
 	tsne_orig <- DimPlot(seuratObj, reduction = "tsne_adt", group.by = "origClusterID", combine = FALSE) + NoLegend()
-	tsne_orig <- tsne_orig + ggtitle("Clustering based on scRNA-seq") + theme(plot.title = element_text(hjust = 0.5))
+	tsne_orig <- tsne_orig  +
```

Comment:
I would check out patchwork's plot_annotation():

https://patchwork.data-imaginist.com/articles/guides/annotation.html

This being said, i think my diagnosis might have been wrong.  This already passed combine=FALSE to DimPlot, so the return should be a ggplot object.  see: https://www.rdocumentation.org/packages/Seurat/versions/3.1.4/topics/DimPlot.

Can you verify what kind of object tsne_orig is?